scripts/md2html/md2html.js

```diff
@@ -384,9 +337,14 @@ for (let l in lines) {
         if (delta>0) delta = 1;
         let prefix = '';
         let newSection = '<section>';
-        if (line.includes('## Version ')) {
+        const m = line.match(/# Version ([0-9.]+)$/);
+        if (m) {
             // our conformance section is headlined with 'Version x.y.z'
             newSection = '<section class="override" id="conformance">';
+            // adjust the heading to be at level 2
+            line = '#' + m[0];
+            delta = 1;
+            heading = 2;
```

Comment:
Why do we have to adjust the heading level here? Why not just put it in the source correctly?

Reply:
That strongly depends on what we consider "correct" in the source:

* existing spec versions

```md
# OpenAPI Specification

#### Version x.y.z
```

* new patch spec versions

```md
# OpenAPI Specification

## Version x.y.z
```

The new spec versions require this change, otherwise the "Version x.y.z" will end up as a top-level heading, not a subheading below "OpenAPI Specification".

Here's the output of the current algorithm, which works with the unaltered old specs and the repaired new specs:
* https://ralfhandl.github.io/OpenAPI-Specification/oas/v3.1.1.html#openapi-specification
* https://ralfhandl.github.io/OpenAPI-Specification/oas/v3.1.0.html#openapi-specification

Reply:
This was a result of the markdown changes which corrected the heading levels